pkg/objectio/block.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2021 Matrix Origin
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package objectio
+
+import (
+	"bytes"
+	"encoding/binary"
+	"github.com/matrixorigin/matrixone/pkg/container/batch"
+)
+
+// Block is the organizational structure of a batch in objectio
+// Write one batch at a time, and batch and block correspond one-to-one
+type Block struct {
+	// fd is the handle of the block
```

Comment:
Does fd belongs here?   I thought block is all about data.    You get fd only when you do I/O.